### DISCUSSION TITLE
Index development updates - Updated mid-March

### DISCUSSION THREAD

**Discussion by aliwhitman** [shcaba/Sebastes_melanops_OR#5] [General]
I have had a pet project this cycle to automate as much as possible the development of ODFW's typical fishery-dependent datasets. I am also working with Kelli Johnson to change the workflow a bit to allow for folks outside of ODFW to assist with the development of any index, after the data have been provided.  Long-term, it would be advantageous to house datasets on Pac/RecFIN but that is for another cycle.  

For three datasets (ORBS - Recreational, At Sea Observer - Recreational, Nearshore logbook - Commercial), I have been: 

- updating internal connections to our new SQL databases
- streamlining the filtering process (code) to be generalized (i.e. not specific to a particular species) and more efficient
- updating documentation that can be fed directly into an assessment document via Latex or markdown file
- working with Kelli J. and Melissa M. to move away from the deltaGLM methodology to more defensible modelling procedures that use objective criteria to select the most appropriate approach from a variety of options (including sdmTMB, which I have been testing with Kelli using ORBS data)

I am hoping to have the connections/filtering portions complete by December, with datasets from all three sources through 2021 to test on modeling approaches in January. Then hopefully, just updating with final 2022 data in February for final inclusion in the models. 

As of right now, I have At Sea and ORBS (connections, filtering) completed and will be starting on Nearshore logbook after Thanksgiving (but there is less work to do for that dataset).

**Comment by shcaba** [comment]
Do you mind filling in the time ranges for all of the indices we are using?
This is the list of what I have:

* Adult tagging index (2002-20??) 
* Adult acoustic survey ()
* SMURFS juvenile index ()

* MRFSS dockside index (1980-2000)
* ORBS dockside index (2001-present)
* Oregon on-board observer index (2001,2003-present)
* Oregon nearshore logbook index ()

Did I miss any?

**Reply by aliwhitman** [reply]
Adult tagging survey - 2002 - 2013 (but excluded 2002 - 2004 on ODFW recommendation) 
Adult acoustic survey - 2021
SMURF jv index (tentative) - 2011 - 2022 (if most recent years able to be ID'd 2019 - 2022, no 2020) 
OR nearshore logbook - 2003 - 2022 

I think that's all of them!

**Comment by aliwhitman** [comment]
Mid-March update: 

- Ali has final, filtered datasets for 1) At Sea observer and 2) ORBS
- Over next week, Ali will be working on 3) nearshore commercial logbook and 4) marine reserves hook and line 
- I also checked in with the SMURF folks to be sure they were aware of the data deadline (in a month!) 
- Ali also reviewed modelling script with Melissa Monk, who will provide a clean version of this by the end of the week 
- Goal is to have all the indices complete by the data deadline! 


- Discussion with Jason today: 
- MRFSS and Pit tagging indices will just be rolled over from last assessment 
- Leif is providing hydroacoustic data/write-up, and Jason is working on adding this to the document
- There was a "small" fish fleet last time around - these were a couple of years of small fish - theoretically, this could be replaced with SMURF data (these were special project samples, so Ali could re-provide them if we want to retain them)
